ktor-server/ktor-server-plugins/ktor-server-csrf/jvmAndNix/src/io/ktor/server/plugins/csrf/CSRF.kt

```diff
@@ -38,6 +38,11 @@ public val CSRF: RouteScopedPlugin<CSRFConfig> = createRouteScopedPlugin("CSRF",
     val headerChecks = pluginConfig.headerChecks
     val onFailure = pluginConfig.handleFailure
 
+    if (!checkHost && allowedOrigins.isEmpty() && headerChecks.isEmpty()) {
```

Comment:
Can you tell me how this code block is related to the onFailure problem?

Reply:
This is the second bullet in the description, to address that an empty configuration doesn't provide any functionality.  There's no real valid option for the default behaviour in this plugin so I'm proposing that we could log a warning when there's no validation defined.

Reply:
Every commit should have a prefix with a YouTrack issue ID. I don't see the ID for the second commit.

Reply:
Also, having a separate PR for an issue is a good practice.

Reply:
Sure, I'll move it to another 👍